bettercli/__init__.py

```
@@ -2,6 +2,7 @@
 from .decorators import pos_option, kw_option
 from .logger import TogglableDebugLogger
 from .cli import CLI
+from . import cl
```

Ruff (0.8.2)

`.cl` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)